datasets/cn/peoples_congress_wikipedia/crawler.py

```diff
@@ -61,7 +61,7 @@ def crawl_item(
 ):
     name_el = input_dict.pop("name")
     reference = name_el.find(".//sup")
-    if reference:
+    if reference is not None:
```

Comment:
I think a little comment here wouldn't hurt, I can't really figure out what's going on from the commit message either.